conda/common/io.py

```diff
@@ -474,9 +507,26 @@ def submit(self, fn, *args, **kwargs):
 class time_recorder(ContextDecorator):  # pragma: no cover
     start_time = None
     record_file = expand(join('~', '.conda', 'instrumentation-record.csv'))
+    total_call_num = defaultdict(int)
+    total_run_time = defaultdict(float)
 
-    def __init__(self, entry_name):
+    def __init__(self, entry_name=None, module_name=None):
         self.entry_name = entry_name
+        self.module_name = module_name
+
+    def _set_entry_name(self, f):
```

Comment:
This is just so we can copy+paste `@time_recorder(module_name=__name__)` everywhere :P